src/app/prevention/strides-against/dialog/dialog.component.html

```diff
@@ -0,0 +1,9 @@
+<div mat-dialog-content>
+  <div style="font-size: 24px">{{data.question}}</div>
+  <mat-divider></mat-divider>
+</div>
+<div style="text-align: center">
+  <button mat-stroked-button (click)="setTrue(); save()">{{language?.true}}</button>
```

Comment:
I think line  6 and 7 can be combined into 1?

Reply:
For that, we'll need a for loop => an array to be made in a component.ts file. I am thinking this might not be worth for an additional line. @shreyans29